src/app/organizations/create-organization/create-organization.component.ts

```diff
@@ -37,11 +37,18 @@ export class CreateOrganizationComponent {
             switchMap(organization => {
                 return this.popupService.new({
                     type: 'success',
-                    title: 'Successfully created a new organization'
+                    title: 'Investment group created!',
+                    text: `Successfully created new investment group.
+                           Verify group creation with your blockchain wallet. You will be prompted now.`,
+                    confirmButtonText: 'Verify group'
```

Comment:
the text and confirm button text is kinda wrong and duplicated:
- text says 'you will be prompted now', but you won't. you will be navigated to the investment group screen and then you will have a popup that will say that 'you will be prompted now'. so, double and misleading text.
- button says 'verify group', but you won't verify the group with this button click. you will be navigated to the investment group screen. that screen will handle what it needs to handle.

I propose just setting the title as `'Investment group created!'`. That's enough. When user clicks OK, it will be navigated further.

Reply:
You're right, task had this in description so I thought it was discussed and agreed upon.
@filipdujmusic are you ok with changes suggested by @mightymatth?

Reply:
Makes sense to me, let's do it the way @mightymatth suggested.